Strip some unnecessary features in dependencies to reduce size and bloat.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -24 +24 @@
-clap = { version = "3", features = ["derive", "env"] }
+clap = { version = "3", default-features = false, features = ["std", "derive", "env"] }
@@ -35,2 +32,2 @@
-env_logger = { version = "0.10.0", default-features = false, features = ["color"] }
-regex = "1"
+env_logger = { version = "0.10.0", default-features = false }
+regex = { version = "1", default-features = false, features = ["std", "perf", "unicode-case", "unicode-gencat", "unicode-perl"] }
